Resources/Prototypes/_Goobstation/Changeling/Entities/Objects/Weapons/Melee/changeling_hammer.yml

```
@@ -11,12 +11,17 @@
+    attackRate: 1.35
     damage:
       types:
         Blunt: 20
         Structural: 50
-    heavyStaminaCost: 12
+    heavyStaminaCost: 10
+    heavyRangeModifier: 1.75
+    heavyRateModifier: 2.5
+    heavyDamageBaseModifier: 1.5
+    maxTargets: 12
+    angle: 150
```

YAMLlint (1.35.1)

[error] duplication of key "angle" in mapping

(key-duplicates)